package.json

```diff
@@ -24,6 +24,7 @@
     "husky": "^8.0.2",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
+    "jest-sorted": "^1.0.14",
```

Comment:
Looks like this config isnt complete

![Screenshot 2023-11-21 at 15 18 55](https://github.com/ELR143/elr143-news-server/assets/150331972/e7f8c74a-7c68-4e66-9378-b51f4e97b732)